components/ILIAS/Init/classes/class.ilInitialisation.php

```diff
@@ -1270,6 +1291,7 @@ protected static function initClient(): void
         self::initGlobal("https", "ilHTTPS", "./components/ILIAS/Http/classes/class.ilHTTPS.php");
         self::initSettings();
         self::setSessionHandler();
+        self::initMail($GLOBALS['DIC']);
```

Comment:
Please don't reintroduce this method, put new registrations to `\ILIAS\Mail\Service\MimeMailService`.